disciplines/chemistry.ttl

```diff
@@ -25,14 +25,14 @@
                                                dcterms:license "https://creativecommons.org/licenses/by/4.0/legalcode" ;
                                                dcterms:publisher "EMMC ASBL" ;
                                                dcterms:title "Chemistry"@en ;
-                                               rdfs:comment """Contacts:
+                                               rdfs:comment "Contacts:
```

Comment:
Multiline literals are encapsulated with triple quotes `"""`. So this should not be changed.

Reply:
I see. With the last commit, all the annotations have been reverted to the previous state.